scripts/test_all.sh

```diff
@@ -62,21 +62,18 @@ run_all_integration_tests() {
 }
 
 run_all_sig_integration_tests() {
```

Comment:
These should be renamed now that they aren't specific to proof of signature or proof of stake.